client/src/components/Cards/EventCard.tsx

```diff
@@ -1,55 +1,58 @@
-import React from 'react'
-import './EventCard.css'
-import DirectionsBusIcon from '@mui/icons-material/DirectionsBus'
+import React, { useState } from 'react'
+import './EventCard.css';
+import DirectionsBusIcon from '@mui/icons-material/DirectionsBus';
 import CalendarMonthIcon from '@mui/icons-material/CalendarMonth'
-import EventTag from '../Buttons/EventTag.tsx'
-import { EventObj, EventTagType } from '../../types/event.ts'
+import EventTag from "../Buttons/EventTag.tsx";
+import {EventObj, EventTagType} from "../../types/event.ts";
+import getTags from "../Buttons/EventTagger.tsx";
 import { Box, Grid, Typography } from '@mui/material'
+import {useTranslation} from "react-i18next";
+import {useLocation} from "react-router-dom";
 
 type EventCardProps = {
-  event: EventObj
+    event: EventObj
 }
 
-const EventCard: React.FC<EventCardProps> = ({ event }: EventCardProps) => {
-  const dayTemp = String(new Date(event.startTime).getUTCDate()).padStart(2, '0')
-  const getVariant = (offset: number = 0): EventTagType => {
-    switch (offset) {
-      case 0:
-        return EventTagType.CULTURE
-      case 1:
-        return EventTagType.FINE_DINING
-      case 2:
-        return EventTagType.FOR_KIDS
-      case 3:
-        return EventTagType.MUSIC
-      case 4:
-        return EventTagType.OUTDOOR
-      case 5:
-        return EventTagType.RELIGION
-      case 6:
-        return EventTagType.SPORTS
-      default:
-        return EventTagType.DEFAULT
+const EventCard: React.FC = (params: { event: EventObj} ) => {
```

Comment:
it should be `const EventCard: React.FC<EventCardProps> = ({ event }: EventCardProps) => {`